modules/rmarkdown/main.nf

```diff
@@ -0,0 +1,92 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+include { dump_params_yml } from "./parametrize"
+
+params.options = [:]
+options        = initOptions(params.options)
+params.parametrize = true
+params.implicit_params = true
+params.meta_params = true
```

Comment:
```suggestion
params.options         = [:]
options                = initOptions(params.options)
params.parametrize     = true
params.implicit_params = true
params.meta_params     = true
```

inb4 @drpatelh ;)

Reply:
You're opening two flame-wars :rofl: 

* About parametrize/parameterize I don't care much, but apparently [both is ok](https://english.stackexchange.com/questions/79847/parametrise-or-parameterise-a-curve). I am happy to take votes :P
* I quite dislike aligned `=` because the create a lot of noise in diffs. But for the sake of consistency, I'll adopt it with gritted teeth. Here's an example why I think it's bad: https://github.com/nf-core/modules/pull/649/files#diff-5a76e5e2d27c76053ced9fdba0a3e5045bafc08de5014b8df3cda826f38730a7L42-**R47**. In that PR, just one line has been added, but the diff shows 6 modified lines.

Reply:
Yeaaaah I know. I also personally don't like it but given 90% of Harshil's review of my PRs were those I thought better to soften the blow ;)